src/main/java/duke/Command.java

```diff
@@ -0,0 +1,26 @@
+package duke;
+
+/**
+ * Represents an abstract Command class.
+ * 
+ * @author Sherman Ng Wei Sheng
+ */
+public abstract class Command {
+
+    /**
+     * Returns true if the command is a programme terminating command.
+     *
+     * @return True if it is a terminating command and false otherwise.
+     */
+    public abstract boolean isExit();
```

Comment:
Similar issue as in AddCommand.class, maybe you need to make the change for Command and its subclasses.

Reply:
EDIT: My bad seems like this is the correct naming convention for boolean methods.